### ISSUE TITLE
Learnings is not a real word

### ISSUE THREAD

**Issue by bazscott** [Widdershin/programmers-oath#55]
Please replace the word "learnings" with "lessons", unless you are talking about _Cultural Learnings of America for Make Benefit Glorious Nation of Kazakhstan_, which I don't think applies in this case :)

**Comment by mo-g** [comment]
Good soundtrack though. Always a favourite at international sporting events.

Throw up a PR, if you've got time? :)

**Comment by Widdershin** [comment]
@bazscott could you please provide a definition of what is or is not a "real word"? It's not clear to me.

**Comment by bazscott** [comment]
@Widdershin I'd argue it's not a real word because you won't find "learnings" in a dictionary. It is an erroneous pluralisation, there is no singular noun "a learning". It's a jargon word that, for some bizarre reason, has started to be used by people instead of the perfectly good word "lessons".

Honestly, I'm not that bothered, It's us that I really love the rest of this programmers oath (thanks for putting it out there). I wouldn't want the good message it sends to be undermined by bad grammar. It's like when people, like, spread the word like around, like, all over the place, it, like, detracts from the, like, message. If you see what I mean? 😄

**Comment by Widdershin** [comment]
Not sure that I agree that it's not a real word, as there is quite a tidy little [wiktionary](https://en.wiktionary.org/wiki/learnings) entry.

> However, from circa 2000 it became a buzzword in business speak, particularly in constructions such as “key learnings” or “apply these learnings”; this was preceded by occasional educational use from 1950s. Some disapprove of this, and it sounds ungrammatical enough to be used as an example of broken English, as in the comedy Borat: Cultural Learnings of America for Make Benefit Glorious Nation of Kazakhstan (2006). Suggested alternatives include lessons learned, “things one learned” – or simply lessons – discoveries, findings, insights, and takeaways.
Most major dictionaries do not label the word as uncountable.

Status as a real world aside, the perceived ungrammaticality is enough for me to support a change. I'm not sure that "learnings" is synonymous for "lessons", but I like wiktionary's suggestion of "lessons learned".

**Comment by CaseyRossSD** [comment]
The only reason that it isn't a legitimate word is that "learning" is a verb, not a noun. You can't have a plural verb, so the problem is that you are applying noun rules to a verb. However, I disagree because the point of it being considered correct is that it is being used as a gerund (a verb used a a noun (ex. Learning is fun!)). In the example, "learning" is used as the noun in the sentence, but noun rules don't technically apply to gerunds. These technicalities are so incredibly minute that I think they can be considered insignificant for the sake of common English.

**Bottom line:**
This discussion shouldn't be about weather or not "learnings" is a legitimate word; instead, it should be about weather or not noun rules can be applied to gerunds... is it worth it?

**Comment by mo-g** [comment]
If it prevents people from opening this ticket every month forever, I'd say yes. ;)

It's a pretty small change, and the version in the PR that's currently waiting approval looks great.

**Comment by Widdershin** [comment]
PR is merged, "learnings" is no more. Thanks everyone for contributing to the discussion.
